oci.go

```diff
@@ -41,6 +41,11 @@ var (
 var cgroupsMemDirPath = "/sys/fs/cgroup"
 
 func containerExists(containerID string) (bool, error) {
+
```

Comment:
Ah no, I got that totally wrong. The behaviour of this function is, huuuum, interesting :)